terraform/lambda/dynamodb.tf

```
@@ -0,0 +1,46 @@
+resource "aws_dynamodb_table" "user-credentials-table" {
+  name         = "user-credentials-table"
```

😬  naming is hard

1) don't include the resource type in the name:
```suggestion
  name         = "user-credentials"
```
2) auth already has a [DynamoDB table called user-credentials](https://github.com/alphagov/di-authentication-api/blob/main/ci/terraform/shared/dynamodb.tf#L2) which does something different.

Ideas: `user-verifiable-credentials`, `user-VCs`, `user-issued-credentials`. Dunno

---

3. should we include the environment name in the resource name to avoid collisions if/when we have multiple environments in the same account?

---

Oh good spot. `user-issued-credentials` sounds like a good shout. I'll update to that.

---

We found previously on Verify that we started with the pattern of using the env as a prefix. We quickly became annoyed by it as it crept everywhere and was difficult to back out. 
It might be easier to add it retrospectively if we find that we're deploying multiple envs to the same account though. Is that likely though?

---

Although looking at it, we already do it here, so perhaps I should prefix the table.

---

Have fixed up if you wouldn't mind taking another look.